.gitignore

```diff
@@ -3,4 +3,6 @@
 .sass-cache/
```

Comment:
`*config.codekit3
.DS_Store
.sass-cache/
node_modules
# Ignore all CSS files in all folders
**/*.css
**/*.css.map
# Don't ignore the root style.css
!style.css
!/style.css`